src/api.js

```diff
@@ -19,6 +19,101 @@ export const LIMITS = {
     min: 50,
     daily: 16972,
     monthly: 42431
+  },
+  GBP: {
+    min: 50,
```

Comment:
Min is USD equivalent. Need to convert these to their respective currency amounts.